controller/templates/confd_settings.py

```diff
@@ -79,3 +79,12 @@
 PRIVATE_EXT_REGISTRY_FLAVOR = '{{ getv "/deis/controller/private_ext_registry/flavor" }}'
 PRIVATE_EXT_REGISTRY_DATA = json.loads('''{{ getv "/deis/controller/private_ext_registry/data" }}'''.replace("\'", '"'))
 {{ end }}
+
+# if defined, then scheduler will distribute the units cross that list.
+# it is a list of anything, the scheduler would pick az = AVAILABILITY_ZONE_LIST[num % len(AVAILABILITY_ZONE_LIST)]
+# which means if list is ['a', 'b'], and it is the 3rd container of the same application,
+# we would do, 3 % 2 = 1 => az = list(1) => az=b
+AVAILABILITY_ZONE_LIST=[]
```

Comment:
What I posted was to show the pattern 
```
value = -init-
if ...: 
     use/modify value
return value
```
related to you comment above about azs.

For here, I said I want to stay coherent with the line just above it. 
We can discuss coding style if you want, and if we agree on something I'd be happy to raise another PR that addresses all the necessary changes, but this is out of the scope of this PR.